src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You have defined the `font-family` and `font-weight` in the `html` selector, but it's more common and advisable to define these in the `body` selector. This is because the `body` tag is the main container for the content that is rendered on the page.